cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -9,39 +9,58 @@
 
    Render an article when given:
 
-   settings:
-
-   settings.heading:            A string with the title of the post.
-   settings.body:               A string with content of post preview/post summary.
-   settings.author:             An array with authors of the post.
-   settings.published_date:     A datetime object with published date.
-
-   settings.image_url:          A string with the url of the thumbnail image.
-   settings.image_alt:          A string with the alt text of the thumbnail image.
-
-   settings.post_category:      An array with the categories for the post. Up to 2.
-   settings.post_tags:          An array with the post tags for the post.
-
-   settings.event_start_dt:     A datetime object with the time of the event.
-   settings.event_stream_link:  A string with livestream link
-   settings.event_venue:        A string with the event venue.
-   settings.event_street:       A string with the event street.
-   settings.event_city:         A string with the event city.
-   settings.event_state:        A string with the event state.
-   settings.event_zip:          A string with the event zip.
-
-   settings.comments_close_date: A datetime object marking the deadline of the comment period.
-   settings.preview_link_text:   A string with the description text of an external link.
-   settings.preview_link_url:    A string with the url of an external link.
-
-   path:                        A string with the path for the homepage of the pagetype
-                                `/about-us/blog/`, `/about-us/newsroom/`
+   post:                          An object with specific article information.
+
+   post.specific_class.__name__:  String used to check for EventPage post type.
```

Comment:
We shouldn't be doing things like this in the template. Just add a method to the class python class.